src/parameters/create_parameters.jl

```diff
@@ -111,6 +111,8 @@ function create_parameter_set(config::AtmosConfig)
         :drag_layer_vertical_extent => :σ_b,
         :kappa_2_sponge => :kappa_2_sponge,
         :held_suarez_minimum_temperature => :T_min_hs,
+        :ocean_surface_albedo => :ocean_surface_albedo,
```

Comment:
```suggestion
        :ocean_surface_albedo => :idealized_ocean_albedo,
```
Given @LenkaNovak's comment, we can rename this within ClimaAtmos without a new release.

Reply:
Thanks! I will change this.